openmetadata-ui/src/main/resources/ui/playwright/e2e/Features/IncidentManager.spec.ts

```diff
@@ -47,6 +48,9 @@ test.describe('Incident Manager', PLAYWRIGHT_INGESTION_TAG_OBJ, () => {
 
     const { afterAction, apiContext, page } = await createNewPage(browser);
 
+    // Todo: Remove this patch once the issue is fixed #19140
+    await revokeTokenFromProfilerBotPage(page);
```

Comment:
We are not just revoking but we are reseting it, can we update the method name here?